vertica_python/vertica/column.py

```diff
@@ -90,7 +90,10 @@ def __init__(self, col):
         self.props = ColumnTuple(col['name'], col['data_type_oid'], None,
                                  col['data_type_size'], None, None, None)
 
-        self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
+        try:
+            self.converter = self.DATA_TYPE_CONVERSIONS[col['data_type_oid']][1]
```

Comment:
DATA_TYPE_CONVERSIONS is a list, you can't use `get` like you would do for a dictionary.